<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KNewBankDlg</class>
 <widget class="QDialog" name="KNewBankDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Institution Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="GroupBox1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Institution Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="7" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>https://</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="TextLabel5">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Telephone/Fax:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="TextLabel3">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Street:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="4">
       <widget class="KLineEdit" name="bankCodeEdit">
        <property name="toolTip">
         <string extracomment="Tooltip for 'Bank Code' field. Include an example of the culture-specific term for 'Bank Code' here, if one is common (https://en.wikipedia.org/wiki/Bank_code).">A bank code assigned to Institution by a central bank, a bank supervisory body or a Bankers Association. Known as 'Routing Number' (US accounts) or a 'Sort Code' (UK accounts).</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="TextLabel7">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>SWIFT/BIC</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="KLineEdit" name="nameEdit"/>
      </item>
      <item row="7" column="2">
       <widget class="KLineEdit" name="urlEdit"/>
      </item>
      <item row="6" column="1" colspan="4">
       <widget class="KLineEdit" name="bicEdit">
        <property name="toolTip">
         <string extracomment="Tooltip for 'SWIFT/BIC' field">A Business Identifier Code, also known as SWIFT.</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="5">
       <widget class="KMessageWidget" name="messageWidget" native="true"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="TextLabel4">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Postal Code:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="4">
       <widget class="KLineEdit" name="telephoneEdit"/>
      </item>
      <item row="3" column="1" colspan="4">
       <widget class="KLineEdit" name="postcodeEdit"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Website</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="4">
       <widget class="KLineEdit" name="streetEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="TextLabel1">
        <property name="text">
         <string comment="institution name column">Name:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="KLineEdit" name="cityEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="TextLabel2">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>City:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="TextLabel6">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Bank Code:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QToolButton" name="iconButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="Spacer30">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QWidget</extends>
   <header>kmessagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>cityEdit</tabstop>
  <tabstop>streetEdit</tabstop>
  <tabstop>postcodeEdit</tabstop>
  <tabstop>telephoneEdit</tabstop>
  <tabstop>bankCodeEdit</tabstop>
  <tabstop>bicEdit</tabstop>
  <tabstop>urlEdit</tabstop>
  <tabstop>iconButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
